provider/testdata/src/lib.rs

```diff
@@ -100,18 +100,6 @@ pub mod versions {
     pub fn icu_tag() -> alloc::string::String {
         alloc::string::String::from(super::metadata::ICUEXPORT_TAG)
     }
-
-    /// Gets the segmenter LSTM tag used as the test data source
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// assert_eq!("v0.1.0", icu_testdata::versions::segmenter_lstm_tag());
-    /// ```
-    #[deprecated(since = "1.3.0", note = "use `compiled_data`")]
```

Comment:
nit: semver?

Reply:
I'm removing this because it wasn't in 1.2